src/Components/BookState.js

```diff
@@ -8,34 +9,34 @@ const BookState = ({
   const dispatch = useDispatch();
 
   return (
-    <div className="container">
-      <div className="details">
+    <div className={styles.container}>
+      <div className={styles.details}>
         <h4>{category}</h4>
         <h3>{title}</h3>
         <p>{author}</p>
         <ul>
           <li>
-            <button className="line" type="button">Comments</button>
+            <button className={styles.line} type="button">Comments</button>
           </li>
           <li>
-            <button className="line" type="button" onClick={() => dispatch(deleteBook(id))}>
+            <button className={styles.line} type="button" onClick={() => dispatch(deleteBook(id))}>
               Remove
             </button>
           </li>
           <li>
-            <button className="button" type="button">Edit</button>
+            <button className={styles.button} type="button">Edit</button>
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up: Make sure to use the same border styles and also follow the spacing so It matches the design and also uses the same font family as mentioned in Zaplin design. The mentioned Zaplin design's font families are check below :point_down:  

Zaplin Font Families:
- Montserrat
- RobotoSlab

Your Font family:
- Poppins

- [ ] So kindly replace all the font families with provided font families and also make sure your whole project looks like Zaplin Design :+1: check below :point_down:  the comparison your design and Zaplin design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-07-28 01-13-31](https://github.com/tchasinga/bookstoring/assets/45226848/60502b1e-101e-477f-8b0e-ff11c59e5a83)

**Figma Design**
![Screenshot from 2023-07-28 01-13-58](https://github.com/tchasinga/bookstoring/assets/45226848/d38f0568-d851-425a-a37e-b556863e8175)


- [ ] If you check you missed one border and also the card is not the same as the Zaplin design so kindly fix it :+1: 
- [ ] If you check your font styles it different from Zaplin design so kindly fix it as well